tools/delete_from_ccx_s3_bucket.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/env python3
+
+import boto3
+import os
+from datetime import datetime, timezone
+
+bucket = os.getenv('AWS_S3_BUCKET')
```

Comment:
can you guard it with ``if __name__ == "__main__"``?
all sort of tools might import this script and do some damage, so it's always a good idea to have it guarded